packages/x-components/src/__stubs__/related-prompts-stubs.factory.ts

```diff
@@ -25,9 +26,10 @@ export function getRelatedPromptsStub(amount = 12): RelatedPrompt[] {
 export function createRelatedPromptStub(suggestionText: string): RelatedPrompt {
   return {
     suggestionText,
-    nextQueries: createNextQueriesArrayStub(10),
+    nextQueriesRelatedPrompts: createNextQueriesArrayStub(10),
     modelName: 'RelatedPrompt',
-    type: Math.random() < 0.5 ? 'CURATED' : 'SYNTHETIC'
+    type: Math.random() < 0.5 ? 'CURATED' : 'SYNTHETIC',
```

Comment:
using `math random ` in a test stub is not very consistent... are we dealing with the type propert somehow? if not, we can just put a static value